src/rgw/rgw_cache.cc

```diff
@@ -637,7 +637,7 @@ size_t DataCache::lru_eviction(){
 }
 
 
-void DataCache::remote_io(struct librados::L2CacheRequest *l2request ) {
+void DataCache::remote_io(struct L2CacheRequest *l2request ) {
```

Comment:
this (and all pointer decls) really should be "L2CacheRequest*";  one object is being declared, of type pointer-to-L2CacheRequest

Reply:
all pointer decls fixed